src/CoreVideo/CVPixelBufferPool.cs

```diff
@@ -159,7 +161,7 @@ public CVPixelBufferPool (NSDictionary poolAttributes, NSDictionary pixelBufferA
 		}
 
 		public CVPixelBufferPool (CVPixelBufferPoolSettings settings, CVPixelBufferAttributes pixelBufferAttributes)
-			: this (settings.GetDictionary (), pixelBufferAttributes.GetDictionary ())
+			: this (settings.GetDictionary () ?? new NSDictionary (), pixelBufferAttributes.GetDictionary () ?? new NSDictionary ())
```

Comment:
why ?

`x.GetHandle` is an extension method that will return `IntPtr.Zero` if `x` is null

Also you're creating two instances of the same, immutable instance of `NSDictionary`.

if `CVPixelBufferPoolCreate` does not accept `null` then it's incorrect and should be using `GetNonNullHandle` extension method